test/system/255-auto-update.bats

```diff
@@ -5,31 +5,37 @@
 
 load helpers
 
+DASHUSER=""
 UNIT_DIR="/usr/lib/systemd/system"
```

Comment:
Not related to your changes but can we please store the unit files in `/run/systemd/system/` for root and `$XDG_RUNTIME_DIR/systemd/user/` for rootless. Using a tmp dir should prevent the files from dangling for ever if the test got interrupted. `/usr/lib/systemd/system` is used by the package manager I do not want some random test to mess with files in there.

Reply:
see `systemd.unit(5)`

Reply:
@Luap99 **thank you**! That has long been a top concern of mine, but I hadn't found that particular section of the systemd man pages. Fixed, and I will follow up separately with a fix for the other tests that use `/lib/systemd`.